test/nocks.ts

```diff
@@ -28,8 +28,8 @@ export type Validator = (body: {
 // TODO: Determine if the type of `validator` is correct.
 export function oauth2(validator?: Validator): nock.Scope {
   validator = validator || accept;
-  return nock('https://accounts.google.com')
-      .post('/o/oauth2/token', validator)
+  return nock(/https:\/\/(accounts\.google\.com|www\.googleapis\.com)/)
```

Comment:
nit: With gcp-metadata@0.6.3 only one of these two urls is possible. You don't need to test either/or. I would suggest dropping the old version.

Reply:
Ok I'll drop the old version (https://accounts.google.com).